users/forms.py

```diff
@@ -5,6 +5,7 @@
 class UserCreationForm(forms.ModelForm):
     password1 = forms.CharField(label="Password", widget=forms.PasswordInput())
     password2 = forms.CharField(label="Password confirmation", widget=forms.PasswordInput())
+    birth_date = forms.DateField(label='Birthday')
```

Comment:
I think that writing it this way has some other side effects, you can change a label of a field of a modelForm by writing 
`label={
             'birth_date': 'Birthday'
}`
or something like that

Reply:
https://stackoverflow.com/questions/36905060/how-can-i-change-the-modelform-label-and-give-it-a-custom-name/36905090

Reply:
I don't see it causing any harm.
If this works there's no reason to update it IMO.

Reply:
I think that it's better to write it the way shown in the stackoverflow page, it's a small change too 🧙

Reply:
But it doesn't really matter if it works

Reply:
It's really doesn't matter. both doing the same.